Fortinet/fortigate/ingest/parser.yml

```diff
@@ -314,6 +318,10 @@ stages:
       - set:
           network.bytes: "{{parsed_event.message.get('sentbyte', 0) | int + parsed_event.message.get('rcvdbyte', 0) | int}}"
         filter: "{{parsed_event.message.get('sentbyte') != None or parsed_event.message.get('rcvdbyte') != None }}"
+      - set:
+          fortinet.fortigate.bitrate: "{{(final.network.bytes | int * 8) // (parsed_event.message.duration | int)}}" # bits per second
+        filter: "{{parsed_event.message.duration | int(-1) > 0}}"
+
```

Comment:
After internal discussions. We considered that isn't recommended to compute of a bitrate inside a parser.
Can you remove this part? Thank you.